### DISCUSSION TITLE
Add useFcmV1 request param

### DISCUSSION THREAD

**Discussion by mgscreativa** [ctwillie/expo-server-sdk-php#42] [Ideas]
As legacy FCM tokens are expiring in favor of new FCM v1 tokens expo notifications server added a new request parameter *useFcmV1*

I managed to get it working like this:
```
$expo = new Expo(
    null, array(
    'useFcmV1' => true,
));
```
Then in ExpoClient
```
$response = $this->client->post(self::EXPO_BASE_URL . '/push/send', [
    'http_errors' => false,
    'headers' => $headers,
    'body' => $body,
    'query' => [
        'useFcmV1' => $this->client->getConfig('useFcmV1') ? 'true' : 'false',
    ]
]);
```

The issue with my implementation is that I'm using the deprecated getConfig getter from guzzle http.

Can you add the new useFcmV1 option the it should be?

Thanks!

**Comment by ctwillie** [comment]
Answered in other discussion [here](https://github.com/ctwillie/expo-server-sdk-php/discussions/41).
